roles/pulp_common/tasks/install.yml

```diff
@@ -14,6 +14,46 @@
         name: '{{ pulp_preq_packages }}'
         state: present
 
+    - name: Package facts
+      package_facts:
+        manager: "auto"
+
+    - name: Packages
+      debug:
+        var: ansible_facts.packages
```

Comment:
We can keep these if you really want but see my existing tasks in roles/pulp_common/tasks/main.yml . I add `verbosity: 1` so users do not see them by default.

Also, they should have better taks names (so users can make sense of the output when they run pulp_installer.)

This comment applies to your new task in roles/pulp_common/tasks/main.yml as well.